lesson2/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="stylesheet" href="style.css" type="text/css" />
+        <script src="index.js" defer></script>
+        <title>TerraceTechフロントエンドエンジニア養成所</title>
+    </head>
+    <body>
+        <ul id="js-target"></ul>
+    </body>
+</html>
```

Comment:
nits
The red circle display on github can be turned off with the following settings.
https://qiita.com/shuhei_fujita/items/960754d0c821f9f6407c